pkgs/by-name/cr/cryptpad/package.nix

```diff
@@ -18,18 +18,55 @@ let
   # nix version of install-onlyoffice.sh
   # a later version could rebuild from sdkjs/web-apps as per
   # https://github.com/cryptpad/onlyoffice-builds/blob/main/build.sh
-  onlyoffice_build =
+  onlyoffice_oldfetch =
     rev: hash:
     fetchFromGitHub {
       inherit rev hash;
       owner = "cryptpad";
       repo = "onlyoffice-builds";
     };
+
+  # New method for v7+ versions that use ZIP releases
+  onlyoffice_newfetch =
+    version: hash:
+    fetchurl {
+      url = "https://github.com/cryptpad/onlyoffice-editor/releases/download/${version}/onlyoffice-editor.zip";
+      inherit hash;
+    };
+
   onlyoffice_install = oo: ''
     oo_dir="$out_cryptpad/www/common/onlyoffice/dist/${oo.subdir}"
-    cp -a "${onlyoffice_build oo.rev oo.hash}/." "$oo_dir"
-    chmod -R +w "$oo_dir"
-    echo "${oo.rev}" > "$oo_dir/.commit"
+    ${
+      if oo ? "version" then
+        ''
+          # New method for v7+ versions
+          mkdir -p "$oo_dir"
+          cd "$oo_dir"
+          unzip ${onlyoffice_newfetch oo.version oo.hash}
+          echo "${oo.version}" > "$oo_dir/.version"
```

Comment:
This is after `cd $oo_dir` so looks a bit odd.. but the file is present:
```
$ cat result/lib/node_modules/cryptpad/www/common/onlyoffice/dist/v7/.version 
v7.3.3.60+11
```
so I must be missing something